### DISCUSSION TITLE
will the v5 frame work on a v4 board?

### DISCUSSION THREAD

**Discussion by mer2329** [sanni/cartreader#471] [Need help]
i had ordered a v4 board by accident (my own fault for not double checking what i clicked on, live and learn am i right?) 
so im wondering will a v4 main board (and 6 slot adapter) fit on the v5 frame?

im assuming it will seeing as the 6 slot adapter for the v5 and v4 are identical (used a checksum comparison toom for the gerbers) and the v4 has the same mounting holes.

main reason i ask is im too lay to fix my printer so im thinking of just ordering the frame from a print house, and the v5 frame is MUCH cheaper then the v4 frame.

**Comment by sanni** [comment]
Yes that will fit. Just remember the V4 main_pcb is missing the 3V/5V switch, so you have to wire one up manually for the six slot adapter.

**Comment by mer2329** [comment]
thanks
ive ordered the gba daughterboard and its my understanding that handles voltage switching.

**Reply by Icesythe7** [reply]
Imo cut ur loses and pay the 8 bucks to get the v5 board the V4 is a huge pita especially with the gba board that you have to bridge
